examples/multiple_with_dns/memorystore.tf

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  version = "~> 2.5.0"
+}
+
+module "memorystore" {
+  source            = "../../"
+  project           = "memorystore"
+  enable_apis       = false
+  managed_zone_name = "my-example-zone"
+
+  instance_configs = [{
+    name                    = "memorystore-2",
+    memory_size_gb          = "1",
```

Comment:
Can we make these optional? Also this should be an int?

Reply:
It's a number type in the variable config, and I don't believe we can make it optional since it's a list of objects and that feature is supported yet.

Reply:
I'll update these lines anyhow.